pkg/minikube/cluster/cluster.go

```diff
@@ -225,6 +229,21 @@ func UpdateCluster(h sshAble, d drivers.Driver, config KubernetesConfig) error {
 		}
 	}
 
+	// loop over .minikube/addons and transfer them to vm
+	searchDir := constants.MakeMiniPath("addons")
```

Comment:
We just discussed making an interface for types of assets to copy into the VM, and having two implementations of it.